libpam/pam_modutil_searchkey.c

```diff
@@ -29,10 +29,28 @@ econf_search_key (const char *name, const char *suffix, const char *key)
 {
 	econf_file *key_file = NULL;
 	char *val;
+	econf_err error = ECONF_SUCCESS;
 
-	if (econf_readDirs (&key_file, VENDORDIR, SYSCONFDIR, name, suffix,
-			    " \t", "#"))
-		return NULL;
+#ifdef HAVE_ECONF_READCONFIG
+	char *parsing_dirs = NULL;
+        if (asprintf(&parsing_dirs, "PARSING_DIRS=%s:/run:%s", VENDORDIR, SYSCONFDIR) < 0) {
```

Comment:
If I'm not mistaken, this changes behavior by adding `/run`, shouldn't it be documented?

Reply:
Yes, I have added a comment in the code, but I am assuming that it is expected on other places too.
Where would you like to have it ?

Reply:
I suppose this should be reflected in the documentation somehow, currently it says only about SYSCONFDIR and VENDORDIR.

Reply:
As pam currently does not write setting temporary to run I think it is better not to parse this directory at all.
The less input data the less risk for "inject" invalid data :-)